docs/content/release-notes/flink-1.15.md

```diff
@@ -406,6 +406,13 @@ updating the client dependency to a version >= 7.14.0 is required due to interna
 The old JDBC connector (indicated by `connector.type=jdbc` in DDL) has been removed.
 If not done already, users need to upgrade to the newer stack (indicated by `connector=jdbc` in DDL).
 
+#### [FLINK-26126](https://issues.apache.org/jira/browse/FLINK-26126)
```

Comment:
I think it would be good to add a separate headline as "Extensible unified Sink uses new metric to capture outgoing records".

Reply:
Excellent! I forgot that.